dictionary.json

```
@@ -15380,7 +15380,7 @@
   {
     "toaq": "sua",
     "type": "predicate",
-    "english": "▯ is necessary for ▯ to happen.",
+    "english": "▯ being the case is necessary for ▯ to happen.",
```

For symmetry’s sake, maybe like this?
```suggestion
    "english": "▯ being the case is necessary for ▯ to be the case.",
```